blockbase/index.php

```diff
@@ -1,20 +1 @@
-<?php
-/**
- * The main template file
- *
- * This is the most generic template file in a WordPress theme
- * and one of the two required files for a theme (the other being style.css).
- * It is used to display a page when nothing more specific matches a query.
- * E.g., it puts together the home page when no home.php file exists.
- *
- * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
- *
- * @package Blockbase
- * @since 1.2.9
- */
-get_header();
-
-// the query
-echo gutenberg_block_template_part( 'index' );
-
-get_footer();
+# This page intentionally left blank
```

Comment:
Should this be a PHP comment?

Reply:
I made it a PHP page now, so the PHP comment fits.

Reply:
I've never knew that you could do single line comments in php with a #!